### PR TITLE
fix: peerDependencies

### PR DESCRIPTION
**Pull request by Scrum**
Errors occur when starting according to the [documentation](https://github.com/vitjs/vit#config)
Error: Cannot find module 'react'
Error: Cannot find module 'globby'

### CODE REVIEW

**Comment by yunsii** [comment]
Thanks for your help. I reviewd my code. 'globby' should be 'dependencies' of `vitjs/vit` package and root project, refer to https://github.com/vitjs/vit/blob/master/packages/vit/src/generateFiles/vit.ts#L6 and https://github.com/vitjs/vit/blob/master/scripts/copy.js#L2

As for 'react', It just used to generate template code and compatiable with `@ant-design/icons` refer to https://github.com/vitjs/vit/blob/master/packages/vit/src/generateFiles/utils.ts#L6

I still not clear of your usage scenarios. And I used in https://github.com/theprimone/pinyin and https://github.com/theprimone/vite-react, it works very well.

**Comment by Scrum** [comment]
> Thanks for your help. I reviewd my code. 'globby' should be 'dependencies' of `vitjs/vit` package and root project, refer to https://github.com/vitjs/vit/blob/master/packages/vit/src/generateFiles/vit.ts#L6 and https://github.com/vitjs/vit/blob/master/scripts/copy.js#L2
> 
> As for 'react', It just used to generate template code and compatiable with `@ant-design/icons` refer to https://github.com/vitjs/vit/blob/master/packages/vit/src/generateFiles/utils.ts#L6
> 
> I still not clear of your usage scenarios. And I used in https://github.com/theprimone/pinyin and https://github.com/theprimone/vite-react, it works very well.

[repository](https://github.com/inside-demo/vite-vit-demo) where reproduced

**Comment by yunsii** [comment]
@Scrum I have fixed this error. You should add `react` to `dependencies` in your repo, this is a react project after all. `globby` dep has fixed by `@vitjs/vit@0.6.1` https://github.com/vitjs/vit/commit/b4751013bd2e7862c265a54a4b549f5c5bfce332 

Thanks for your feedback.

**Comment by Scrum** [comment]
> You should add `react` to `dependencies` in your repo, this is a react project after all

`peerDependencies` gives a clear understanding of what packages are needed to work you don't have to include them as a dependency

File: packages/vit/package.json [hunk 1]
```diff
@@ -45,5 +45,9 @@
     "@types/mustache": "^4.1.1",
     "react": "^17.0.2",
     "vite": "^2.1.3"
+  },
+  "peerDependencies": {
+    "globby": "^11.x",
+    "react": "^17.x"
```
Comment (yunsii) [inline packages/vit/package.json]: Due to `globby` dep had fixed, `globby` can remove from `peerDependencies`. `react: "16.x || 17.x"` should be better?

**Review by yunsii — COMMENTED** [review]
> peerDependencies gives a clear understanding of what packages are needed to work you don't have to include them as a dependency

I have never used `peerDependencies`, thank you for your suggestion 😂

File: packages/vit/package.json (hunk 1 above)
Comment (Scrum) [reply packages/vit/package.json]: Yeah, so much better )

**Pull request merged**